src/pages/LoginPage.js

```diff
@@ -0,0 +1,51 @@
+import React,{useState} from 'react';
+import {Link,useNavigate} from "react-router-dom";
+
+function LoginPage() {
+
+    const [email,setEmail]=useState("");
+    const [password,setPassword]=useState("");
+    const navigate=useNavigate();
+
+    const fetchUser=()=>{
+      if(email && password){
```

Comment:
make use of api call later, if email and password are in backend users.js

Reply:
yes sure